rust/ql/src/utils/modelgenerator/CaptureMixedSummaryModels.ql

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
I'm not sure we'll want this query, I hope that `CaptureContentSummaryModels.ql` should be enough for us.

Reply:
I don't have strong feelings but I'd be inclined to leave the files in places because:
1/ The shared Python script assumes that these files are there.
2/ For consistency with C# and Java
3/ We might never have to ever use them, but perhaps it could come in handy for debugging or experimenting through the generation script.